resources/generate_book.py

```diff
@@ -659,6 +659,62 @@ def make_book(rm: ResourceManager, i18n: I18n, local_instance: bool = False):
+            clay_knapping('tfc:clay_knapping/spindle_head', 'The $(thing)Unfired Spindle Head$() is knapped from clay. It can then be $(l:mechanics/heating)fired$() to make a $(thing)Spindle Head$(). To complete the spindle, craft it with a $(thing)Stick$().'),
+            crafting('tfc:crafting/wool_yarn', text_contents='Crafting $(thing)Wool$() with a Spindle yields $(thing)Wool Yarn$().'),
+            crafting('tfc:crafting/wood/acacia_loom', text_contents='The loom is crafted from just $(thing)Lumber$() and a $(thing)Stick$().'),
+            loom_recipe('tfc:loom/wool_cloth', 'The recipe for $(thing)Wool Cloth$() takes 16 $(thing)Wool Yarn$(). Adding to the loom is done with $(item)$(k:key.use)$(). Then, hold down $(item)$(k:key.use)$() to begin working the loom.'),
+            image('tfc:textures/gui/book/tutorial/loom_full.png', text_contents='The loom after being filled with $(thing)Wool Yarn$().'),
+            image('tfc:textures/gui/book/tutorial/loom_working.png', text_contents='The filled loom being worked by holding $(item)$(k:key.use)$(). When complete, $(item)$(k:key.use)$() retrieves the item.'),
+            loom_recipe('tfc:loom/wool_block', '$(thing)Wool Cloth$() can be re-woven into $(thing)Wool Blocks$(). Wool blocks can be dyed.'),  # todo: ref dyeing page
+            loom_recipe('tfc:loom/silk_cloth', '$(thing)Silk Cloth$() can be made in the loom out of $(thing)String$(). It can be used as a wool cloth substitute in some cases.'),
+            loom_recipe('tfc:loom/burlap_cloth', '$(thing)Burlap Cloth$() does not have a use, but it can be made from $(l:mechanics/crops#jute)Jute Fiber$().'),
+            text('There are two main uses for cloth: $(thing)Beds$() and $(thing)Paintings$().'),
+            crafting('tfc:crafting/vanilla/color/light_blue_bed', 'tfc:crafting/vanilla/painting'),
+        )),
+        entry('bread', 'Bread', 'tfc:food/barley_bread', pages=(
+            text('Bread is the processed form of the various grain crops, such as $(l:mechanics/crops#barley)Barley$(). Breaking a grain crop just drops an item similar to vanilla wheat, which is not useful on its own. It must be processed into $(thing)Bread$(), which can then be eaten or used in $(l:mechanics/sandwiches)Sandwiches$().'),
+            crafting('tfc:crafting/barley_cutting', text_contents='First, cut the straw off of the food with a $(thing)Knife$().'),
+            item_spotlight('tfc:food/rye_grain', text_contents='Grains are the longest-lasting stage of the process, decaying much slower than most foods. Preserving grain is best done by leaving them in this state until it is time to proceed!'),
```

Comment:
"Preserving grain is best done by leaving them in this state until it is time to proceed!" -> "As a result they can be stored for <> without expiring."

Otherwise this reads like an opinion - tell the player the facts, let them make a decision, rather than telling them what to do.

Replace <> with the actual amount of time that grain decays in in a vessel (I think it's 2 months without vessel so 4 months with?)